assets/blocks/course-list-block/course-list.scss

```diff
@@ -49,3 +49,49 @@ $block: '.wp-block-sensei-lms-course-list';
 		padding: 1.25em;
 	}
 }
+
+.featured-badge {
+	display: none;
+}
+
+
+.featured-image-wrapper {
+	position: relative;
+	.featured-badge {
+		display: block;
+		position: absolute;
+		top: 10px;
+		left: 10px;
+		background-color: #1d2327;
+		padding: 3px 6px;
+		text-decoration: none;
+		border-radius: 2px;
+		white-space: nowrap;
+		z-index: 1000;
+	}
+	.featured-badge-edit {
```

Comment:
I can't find where this class is being used. Can it be removed?

Reply:
22d62b23731f72dafe6962e8be560403ad995c1b